src/views/Game/Game.tsx

```diff
@@ -2516,6 +2516,11 @@ export class Game extends React.PureComponent<GameProperties, any> {
                         </span>
                     }
                 </div>
+                <div className='annulled-indicator'>
+                    { state.annulled &&
+                        'GAME ANNULLED'
```

Comment:
Any text we display to the user needs to be translatable, to do that simply wrap it with an `_` or `pgettext` call, such as `_("Game annulled")` or `pgettext("DIsplayed to the user when the game is annulled", "Game annulled")` (the first argument is a comment string for the translators)

I like the strike through on the profile page, on the game history, but I think I don't like it on the game page.

And last but not least, lets `text-align: center` and remove the all caps, make it just Game Annulled` on the game page

Reply:
Scratch the style change comments, my css didn't correctly, I see it was centered (and red), and with that style the strike through looks good too.

So, just the translation thing, ignore the rest! :)

Reply:
Ah, yes I saw the i18n support, but wasn't sure how to use it.  Updates incoming!